src/Entity/Content.php

```diff
@@ -46,7 +50,7 @@ class Content implements \JsonSerializable
     /**
      * @var string
      *
-     * @ORM\Column(type="string", length=191, name="contenttype")
+     * @ORM\Column(type="string")
```

Comment:
Removing the `length=191` will break on MySQL 5.6. See https://github.com/symfony/symfony-docs/issues/6398

Reply:
Thanks, didn't know that! But this happens only on columns with `unique=true`, right? https://github.com/symfony/symfony-docs/pull/10055/files

Reply:
AFAIK it happens on all columns that are either `unique`, used as an index or as a Foreign Key.

Reply:
fixed